From 93b0c9d52ad7dc218a2aff741504a0a3061b8fe1 Mon Sep 17 00:00:00 2001
From: Ratan Gupta <ratagupt@in.ibm.com>
Date: Fri, 21 Sep 2018 10:24:17 +0530
Subject: [PATCH 1/1] DO-NOT-MERGE: SNMP Client Interface to be backward
 compatible

Latest master having the SNMP client interface which has two parameter
1) IPaddress 2) port.
In GHE we are using three parameter client interface
1) IPaddress 2) port 3) addressType.
so this commit is introduced to have the backward compatibility.

Change-Id: Iccd489081b2049beaf3998cd1013121a6f2e7518
Signed-off-by: Ratan Gupta <ratagupt@in.ibm.com>
---
 xyz/openbmc_project/Network/Client.interface.yaml | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/xyz/openbmc_project/Network/Client.interface.yaml b/xyz/openbmc_project/Network/Client.interface.yaml
index 8c8eaeb..7b5acb8 100644
--- a/xyz/openbmc_project/Network/Client.interface.yaml
+++ b/xyz/openbmc_project/Network/Client.interface.yaml
@@ -17,3 +17,14 @@ properties:
           The value of this property is the network port number.
       errors:
         - xyz.openbmc_project.Common.Error.InternalFailure
+    - name: AddressFamily
+      type: enum[self.IPProtocol]
+      description: >
+          IP protocol type which can be IPv4 or IPv6 etc.
+enumerations:
+    - name: IPProtocol
+      description: >
+          Possible IP protocol types.
+      values:
+        - name: IPv4
+        - name: IPv6
-- 
1.8.3.1

